src/table/tableview/table-view.component.js

```diff
@@ -3,6 +3,7 @@ angular.module('patternfly.table').component('pfTableView', {
     config: '<?',
     dtOptions: '<?',
     colummns: '<',
+    colvis: '<',
```

Comment:
I don't think I agree with using colvis to update the visibility of columns. For one thing, how would I set the table up initially? How would I set 2 or more columns to be not visible?